NDB.Covid19/NDB.Covid19/PersistedData/LocalPreferencesHelper.cs

```diff
@@ -50,6 +50,8 @@ public static void SetIsBackgroundActivityDialogShowEnableNewUser(bool isBackgro
             _preferences.Set(PreferencesKeys.LAST_BACKGROUND_ACTIVITY_DIALOG_SHOW_STATE_NEW_USER, isBackgroundActivityDialogShowEnableNewUser);
         }
 
+        public static bool IsAppLaunchedToShowDialog { get; set; }
```

Comment:
move this to another class. this one is in PersistedData namespace so it does not fit here :)

Reply:
Yes, that's right I will move it.